src/views/GroceryListView/GroceryListView.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import AppContext from '../../context';
+import List from '../../components/List/List';
```

Comment:
Use `index.js` files instead to make clear to external users what the exposed values are.